[2025-07-15T22:04:33.606+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-15T22:04:33.632+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Projeto_Joao_Motta.etapa_2 manual__2025-07-15T22:04:25.116509+00:00 [queued]>
[2025-07-15T22:04:33.642+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Projeto_Joao_Motta.etapa_2 manual__2025-07-15T22:04:25.116509+00:00 [queued]>
[2025-07-15T22:04:33.643+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2025-07-15T22:04:33.658+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): etapa_2> on 2025-07-15 22:04:25.116509+00:00
[2025-07-15T22:04:33.668+0000] {standard_task_runner.py:63} INFO - Started process 59 to run task
[2025-07-15T22:04:33.676+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Projeto_Joao_Motta', 'etapa_2', 'manual__2025-07-15T22:04:25.116509+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/DAG 3.py', '--cfg-path', '/tmp/tmpsissesjy']
[2025-07-15T22:04:33.682+0000] {standard_task_runner.py:91} INFO - Job 3: Subtask etapa_2
[2025-07-15T22:04:33.756+0000] {task_command.py:426} INFO - Running <TaskInstance: Projeto_Joao_Motta.etapa_2 manual__2025-07-15T22:04:25.116509+00:00 [running]> on host 6f67326087ce
[2025-07-15T22:04:34.131+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Projeto_Joao_Motta' AIRFLOW_CTX_TASK_ID='etapa_2' AIRFLOW_CTX_EXECUTION_DATE='2025-07-15T22:04:25.116509+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-15T22:04:25.116509+00:00'
[2025-07-15T22:04:34.132+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-15T22:04:34.404+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-15T22:04:34.405+0000] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/DAG 3.py", line 63, in etapa_2
    df_final.to_excel(arquivo_saida, index=False)
  File "/home/airflow/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 2345, in to_excel
    formatter.write(
  File "/home/airflow/.local/lib/python3.10/site-packages/pandas/io/formats/excel.py", line 946, in write
    writer = ExcelWriter(  # type: ignore[abstract]
  File "/home/airflow/.local/lib/python3.10/site-packages/pandas/io/excel/_openpyxl.py", line 57, in __init__
    from openpyxl.workbook import Workbook
ModuleNotFoundError: No module named 'openpyxl'
[2025-07-15T22:04:34.426+0000] {taskinstance.py:1205} INFO - Marking task as FAILED. dag_id=Projeto_Joao_Motta, task_id=etapa_2, execution_date=20250715T220425, start_date=20250715T220433, end_date=20250715T220434
[2025-07-15T22:04:34.447+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 3 for task etapa_2 (No module named 'openpyxl'; 59)
[2025-07-15T22:04:34.498+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-07-15T22:04:34.549+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-15T22:04:34.552+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
